go-packages/meep-model/model.go

```diff
@@ -36,19 +36,22 @@ var DbAddress = "meep-redis-master.default.svc.cluster.local:6379"
 var redisTable = 0
 
 const (
-	NodeTypeScenario     = "SCENARIO"
-	NodeTypeOperator     = "OPERATOR"
-	NodeTypeOperatorCell = "OPERATOR-CELLULAR"
-	NodeTypeZone         = "ZONE"
-	NodeTypePoa          = "POA"
-	NodeTypePoaCell      = "POA-CELLULAR"
-	NodeTypeUE           = "UE"
-	NodeTypeFog          = "FOG"
-	NodeTypeEdge         = "EDGE"
-	NodeTypeCloud        = "DC"
-	NodeTypeUEApp        = "UE-APP"
-	NodeTypeEdgeApp      = "EDGE-APP"
-	NodeTypeCloudApp     = "CLOUD-APP"
+	NodeTypeScenario              = "SCENARIO"
+	NodeTypeOperator              = "OPERATOR"
+	NodeTypeOperatorCell          = "OPERATOR-CELLULAR"
+	NodeTypeZone                  = "ZONE"
+	NodeTypePoa                   = "POA"
+	NodeTypePoaCellularDeprecated = "POA-CELLULAR"
```

Comment:
Suggest moving this deprecated definition to validator file because it is the only one that uses it. The Model assumes a validated scenario and therefore does not need/use the deprecated fields. Also, any future changes should not impact the previous validation code.

Reply:
moved